src/org/ssgwt/client/ui/form/spinner/DurationSpinner.java

```diff
@@ -164,6 +181,7 @@
         public void onSpinning(long value) {
             if (getHourSpinner() != null) {
                 getHourSpinner().setValue(value, false);
+                oldHourValue = (int)value;
```

Comment:
value, if you look at onSpinning signiture, is a long. The reason being that this field is only used for hours. Minutes is a separate spinner next to this one.